pkgs/by-name/js/json-search/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib
+  , stdenv
+  , fetchFromGitHub
+  , json_c
+  , pkg-config
+}:
```

What uglification was this?

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, json_c
, pkg-config
}:
```

---

I format args like this on all the packages I create, it’s more readable. And quite honestly, It’s your proposal that I find ugly… But since this is not the point and it seems important to you, I’ll change it.